CMSgov/design-system
# [WNMGDS-2206] Theme color page updates

## Summary

https://jira.cms.gov/browse/WNMGDS-2206

### Visual changes
- Removed the additional colors section that contained 2 colors one of which was transparent 
- Updated styling of colors to be presented more like the system colors page 
- Updated styling of utility > text-color page 
- Updated styling of utility > background-color page

### Code changes
- Reorganized existing color-presentation components into the following components:
    - `ColorSwatch`
        - It's just a svg circle with the color
        - Used in the text-color page and the component pages
    - `ColorExampleRow`
        - It's a row with a background color of the specified color plus two text/code elements with the name of the color and the value of the color
        - Used in both the `ColorExampleList` and `ColorRamps` component
    - `ColorExampleList`
        - It's more generic than `ColorRamps`
        - Used on both the theme-colors page and the background-colors page
    - `ColorRamps`
        - A more specialized form of color example that is specific to the system colors page

## How to test

1. Run `yarn start` and visit the theme colors page and see that the additional colors section is no longer on the page at the bottom and the color samples are now rectangular that stretch the page width.

## Checklist




### If this is a change to code:


### If this is a change to documentation:
